backend/coins/eth/coin_test.go

```diff
@@ -35,38 +41,46 @@ func TestCoin(t *testing.T) {
 		nil,
 		nil,
 	)
+}
 
-	require.Equal(t,
+func TestSuite(t *testing.T) {
+	suite.Run(t, new(testSuite))
```

Comment:
maybe run the suite multiple times, once with ETH and once with an ERC20 token /with different number of decimals)? then the SetAmount test also covers tokens and `isFee = true` behaves differently.

Reply:
Ok! I'll push an update